### PR TITLE
enhance(wizard): update translations, refine icons, and add automated…

### PR DESCRIPTION
**Pull request by loduis**
… requirement verification

### CODE REVIEW

File: package.json
Comment (mchev) [inline package.json:1]: Why runtime-core and flatpickr are installed if not used in the modifications ?

Comment (loduis) [reply package.json:1]: After a pnpm install and pnpm run dev
image

You can search by flatpickr

image


The line must be changed, and remove from package.json

import { computed, ref } from '@vue/runtime-core'

image

Comment (mchev) [reply package.json:1]: vue-flatpickr-component is installed and already requires flatpickr as dependencie

Comment (loduis) [reply package.json:1]: It appears to be an issue with the package manager pnpm, which is not resolving flatpickr correctly. With npm work fine

**Pull request merged**